.github/workflows/e2e_call.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         env:
           DOCKER_ID: ${{ secrets.DOCKER_ID }}
           MY_DOCKER_PASSWORD: ${{ secrets.MY_DOCKER_PASSWORD }}
-        run: curl "https://raw.githubusercontent.com/hpi-schul-cloud/end-to-end-tests/main/scripts/ci/fetch.github.sh" | bash -s ${{ matrix.cases }}
+        run: curl "https://raw.githubusercontent.com/hpi-schul-cloud/end-to-end-tests/BC-7461-mongo6/scripts/ci/fetch.github.sh" | bash -s ${{ matrix.cases }}
```

Comment:
this was added only so that the e2e tests called from server repo pass
it should be removed before merge